src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,65 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { Filter } from '../../types/Filter';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  selectedFilter: Filter;
+  onFilterChange: (filter: Filter) => void;
+  searchValue: string;
+  onSearchChange: (phrase: string) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter = ({
+  selectedFilter,
+  onFilterChange,
+  searchValue,
+  onSearchChange,
+}: Props) => {
+  const handleFilterChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
According to the naming conventions for methods in React, it is recommended to name event handlers as 'onEvent'. So, you may want to rename `handleFilterChange` to `onFilterChange`. This will make your code more consistent and easier to understand.

Reply:
export const TodoFilter = ({
  selectedFilter,
  onFilterChange,
  searchValue,
  onSearchChange,
}: Props) => {
  const handleFilterChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
    const newFilter = event.target.value as Filter;

    onFilterChange(newFilter);
  };